NSS-Day-Cohort-49/giffygram-success-kids
# Lam gif

#### Changes Made
1. Added file `sentMessagesPage and receivedMessagesPage with corresponding css modules and styling` to `styles and friends` directory.
1. Modified file `sentMessagesPage and receivedMessagesPage` to include `css styling and functionality to display sent, received, and all messages in messages history page` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user clicks on the messages history button and the sent/all/received messages buttons, then the corresponding messages should display and when a direct message is sent, it should go to the recipient and if credentials don't match database at login, then an alert window should display to retype credentials to log in"
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
